src/oops/SOLID/openClosePrinciple/before/employees/Employee.java

```diff
@@ -39,6 +39,11 @@ public int getMonthlyIncome() {
         return monthlyIncome;
     }
 
+    public int getYearlyIncome() {
+        //assuming constant income for 12 months
+        return getMonthlyIncome() * 12;
+    }
+
```

Comment:
This could have been calculated in InternTaxCalculator in the business logic.

Reply:
Although I agree that this is subjective, the intension here was to keep it at the employee level as this could get complicated later and should be kept at one central place.